tests/test_quickbooks_discovery.py

```diff
@@ -7,7 +7,9 @@
 
 class TestQuickbooksDiscovery(TestQuickbooksBase):
     def name(self):
-        return "tap_tester_quickbooks_combined_test"
+        # Quickbooks uses the token chaining to get the existing token which requires all tests to have same name
+        # So do not overwrite the test name below
```

Comment:
Use docstring like this :
"""
Quickbooks uses the token chaining to get the existing token which requires all tests to have same name
So do not overwrite the test name below
"""